pkg/app/web/src/modules/deployments.ts

```diff
@@ -109,45 +111,51 @@ const convertFilterOptions = (
     statusesList: options.status
       ? [parseInt(options.status, 10) as DeploymentStatus]
       : [],
-    maxUpdatedAt: options.maxUpdatedAt,
+    // NOTE: Now that we're using `cursor`, we don't need this field anymore, so we'll pass 0.
+    // This field will be supported by the filter and may be used in the future.
+    maxUpdatedAt: 0,
```

Comment:
I plan to remove this `maxUpdatedAt` by updating the ListDeploymentRequest proto later, will make other PR for this 🙏